src/screens/Customers/CustomerProfile.tsx

```diff
@@ -36,14 +37,63 @@ const styles = (theme: Theme) =>
   });
 
 interface CustomerProps extends RouteComponentProps {
+  match: { params: string; isExact: boolean; path: string; url: string; }
   classes: { root: string; content: string; header: string; button: string; buttonSecondary: string };
+  siteName: string;
   location: any;
 }
 
 function CustomerProfile(props: CustomerProps) {
   const { classes, match } = props;
-  const customer: CustomerRecord = props.location.state.customer;
+  const customer = props.location.state.customer;
 
+  // helper functions for getting data
+  const isCurrentReading = (mr: MeterReadingRecord): boolean => {
```

Comment:
These looks like they'd be generally useful in the future outside of this component. If helper functions are potentially useful outside of this screen / component, create a utils file (like a `customerUtils` where you have functions that take in a `customerRecord` and give you certain things like their currentMeterReading, etc... a `meterReadingsUtil` file might also be helpful) for it and put all the functions in there and import them to use in here!